Form-Controls/styles.css

```diff
@@ -0,0 +1,70 @@
+{
+    margin:0;
+    padding:0
+}
+.body{
+    background-position: center;
+      background-size: cover;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+      margin-top: 30px;
+      font-size: 15px;
```

Comment:
It's rarely a good idea to set font sizes at less than 16px for body copy. Your site may actually get downranked by Google for doing this.